packages/schemas/alterations/next-1685691718-domain-unique.ts

```diff
@@ -0,0 +1,20 @@
+import { sql } from 'slonik';
+
+import type { AlterationScript } from '../lib/types/alteration.js';
+
+const alteration: AlterationScript = {
+  up: async (pool) => {
+    await pool.query(sql`
+      alter table domains drop constraint domains__domain;
+      create unique index domains__domain ON domains using btree (tenant_id,domain);
+    `);
+  },
+  down: async (pool) => {
+    await pool.query(sql`
+      alter table domains drop constraint domains__domain;
+      create unique index domains__domain ON domains using btree (domain);
```

Comment:
drop index and create constrant?

Reply:
It is unable to drop "index" if the table has data, drop constraint works

Reply:
<img width="906" alt="image" src="https://github.com/logto-io/logto/assets/5717882/b77e9466-5d7a-4744-a9c7-b9c1f201e154">

Reply:
i was blinding by the light